implementations/rust/ockam/ockam_api/src/cli_state/identities.rs

```diff
@@ -85,6 +85,11 @@ impl IdentityState {
     pub fn name(&self) -> &str {
         &self.name
     }
+
+    pub fn is_enrolled(&self) -> bool {
```

Comment:
@etorreborre i set up an `is_enrolled` function directly on `IdentityState` instead of having to navigate to the `is_enrolled` field. lmk if you have any flags against this implementation!

Reply:
This is a very good idea!